openmetadata-service/src/main/java/org/openmetadata/service/security/CatalogPrincipal.java

```diff
@@ -16,14 +16,14 @@
 import java.security.Principal;
 import lombok.Getter;
 
-public record CatalogPrincipal(@Getter String name, @Getter String email) implements Principal {
-  @Override
-  public String getName() {
-    return name;
-  }
+@Getter
+public class CatalogPrincipal implements Principal {
+  private final String name;
+  private final String email;
 
-  public String getEmail() {
-    return email;
+  public CatalogPrincipal(String name, String email) {
+    this.name = name.toLowerCase();
```

Comment:
Why do we want the name to be lowercase?

Reply:
in postgres TestUser1 and testuser1 are both different, for us this is a same user